Library/Homebrew/.yardopts

```diff
@@ -7,6 +7,7 @@
 --template-path yard/templates
 --exclude test/
 --exclude vendor/
+--exclude yard/
```

Comment:
Excluding this, since the top-level `init` and `internal` methods aren't useful to include in the docs (I also verified that this doesn't do anything unexpected when generating yard docs).